docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -0,0 +1,315 @@
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _2025_concept_model_vivarium_alzheimers:
+
+===============================================
+Alzheimer's Disease Early Detection Simulation
+===============================================
+
+.. contents::
+  :local:
+
+.. list-table:: Abbreviations
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+  * - AD
+    - Alzheimer's Disease
+  * - BBBM
+    - Blood-Based Biomarkers
+  * - CSF
+    - Cerebrospinal Fluid
+  * - MCI
+    - Mild Cognitive Impairment
+  * - PET
+    - Positron Emission Tomography
+  * - DALYs
+    - Disability-Adjusted Life Years
+  * - CSU
+    - Client Services Unit
+  * - FHS
+    - Future Health Scenarios
+
+1.0 Overview
+++++++++++++
+
+This project leverages IHME's simulation capabilities to quantify health
+and economic impacts associated with early detection and treatment of
+pre-clinical Alzheimer's disease (AD). The simulation evaluates scenarios
+involving blood-based biomarker (BBBM) testing and a hypothetical
+intervention that slows disease progression.
+
+**Basic Goals:**
+
+- Simulate the patient journey from identification through intervention
+  and outcomes
+- Compare health and economic impacts across reference and alternative
+  scenarios in 10 locations
+
+**Funding and collaboration:**
+
+We are designing this simulation in conjunction with IHME's Client
+Services Unit (CSU) with a focus on health and economic impact. Our team
+will focus on simulating the the health impacts of preclinical AD
+testing and the hypothetical intervention, and the Resource Tracking
+team will use our results to estimate the economic impacts. We will be
+using population forecasts from the Future Health Scenarios (FHS) team.
+
+2.0 Modeling Aims and Objectives
++++++++++++++++++++++++++++++++++
+
+The primary goal is to simulate the impact of early detection strategies
+for Alzheimer's disease using blood-based biomarkers and subsequent
+interventions. The simulation tracks simulants through health states
+from age ~30 to 125 years (or death), capturing progression through
+preclinical AD, mild cognitive impairment (MCI) due to AD, and three
```

Comment:
maybe we don't need MCI (!)
```suggestion
preclinical AD and three
```

Reply:
Good call.  Based on today's discussion, I think we might be able to succeed without separating mild, moderate, and severe stages of clinical AD!

Reply:
Hmm, your comment made me think of a reason we *would* benefit from having the severity modeled --- delaying onset of disease will shift the balance with competing mortality and reduce the years lived with moderate and severe disease.

Reply:
Hmm we should think about how much of that we will be able to get from just having them start AD overall later. If its all or most it might be worth it to simplify.

Reply:
Ok, I think it's still an open question to figure out exactly which stages of AD we want to model explicitly. I'm not going to make any changes to the concept model at this point since we're still unsure.